simulate/simulate_test.go

```diff
@@ -36,15 +36,30 @@ var MutateSeqTests = []struct {
 	{"debug.fasta", 0.5, "debug.gpd"}, //branch length of 1 gives higher chance of returning a new base so you can see a difference even with a short sequence
```

Comment:
Doesn't have to be apart from this checkin, but I think there is a convention in golang to have a folder called testdata and have these miscellaneous files in that folder. Line 36 would then turn into {"testdata/debug.fasta", 0.5, "testdata/debug.gpd"}